internal/form/handler.go

```diff
@@ -25,47 +25,80 @@ type Request struct {
 }
 
 type Response struct {
-	ID             string     `json:"id"`
-	Title          string     `json:"title"`
-	Description    string     `json:"description"`
-	PreviewMessage string     `json:"previewMessage"`
-	Status         string     `json:"status"`
-	UnitID         string     `json:"unitId"`
-	LastEditor     string     `json:"lastEditor"`
-	Deadline       *time.Time `json:"deadline"`
-	CreatedAt      time.Time  `json:"createdAt"`
-	UpdatedAt      time.Time  `json:"updatedAt"`
+	ID             string               `json:"id"`
+	Title          string               `json:"title"`
+	Description    string               `json:"description"`
+	PreviewMessage string               `json:"previewMessage"`
+	Status         string               `json:"status"`
+	UnitID         string               `json:"unitId"`
+	OrgID          string               `json:"orgId"`
+	LastEditor     user.ProfileResponse `json:"lastEditor"`
+	Deadline       *time.Time           `json:"deadline"`
+	CreatedAt      time.Time            `json:"createdAt"`
+	UpdatedAt      time.Time            `json:"updatedAt"`
 }
 
 // ToResponse converts a Form storage model into an API Response.
 // Ensures deadline is null when empty/invalid.
-func ToResponse(form Form) Response {
+func ToResponse(form Form, unitName string, orgName string, editor user.User, emails []string) Response {
 	var deadline *time.Time
+	var editorName string
+	var editorUsername string
+	var editorAvatarURL string
+
 	if form.Deadline.Valid {
 		deadline = &form.Deadline.Time
 	} else {
 		deadline = nil
 	}
 
+	if editor.Name.Valid {
+		editorName = editor.Name.String
```

Comment:
The `String` field of `pgtype.Text` defaults to an empty string when unset. Since the `Response` doesn’t define any other default value or use null/omitempty, the `if pgtype.Text.Valid` check here is unnecessary.